dune-project

```diff
@@ -1,10 +1,14 @@
-(lang dune 2.5)
+(lang dune 2.9)
 (name coq)
-(using coq 0.2)
+(using coq 0.3)
 
 (formatting
  (enabled_for ocaml))
 
+; This gives us full control over the C stubs flags, otherwise
+; dune will add a standard set of flags to all stubs.
+(use_standard_c_and_cxx_flags false)
```

Comment:
Is this related to https://github.com/coq/coq/pull/14519 ?

Reply:
Yup, this is due to the bump of dune version, so orthogonal.